packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -655,6 +659,18 @@ export class GarbageCollector implements IGarbageCollector {
         this.referencesSinceLastRun.set(fromNodeId, outboundRoutes);
     }
 
+    private expireAtTime(expiryMs: number) {
```

Comment:
We should move this out of the class for now and later make this a generic function in one of our common utils. You can call this `safeTimeout` or `longTimeout` similar to how its done here - https://mrcoles.com/maximum-delay-settimeout.

For now, you can keep this function in this file itself (not in the class) and create and issue to make this a generic util since I can see this needed for other similar usages.

Reply:
Here's that issue: https://github.com/microsoft/FluidFramework/issues/9062

Reply:
Also, maybe rename this to `safeSetTimeout` or `setBiggerTimeout`.